spec/twitter_client_spec.rb

```diff
@@ -132,12 +132,52 @@
       response.data["ids"].size.should be > 0
     end
 
-    it "can upload media" do
-      url = "http://staging-careerarc-com.s3.amazonaws.com/test/twitter_card.jpg"
-      response = config.client.upload_media(url)
-      response.should be_successful
-      response.data['media_id'].should be_a_kind_of Numeric
-      response.data['media_id_string'].should be_a_kind_of String
+    context "uploading_media" do
+      let(:url)    { "http://staging-careerarc-com.s3.amazonaws.com/test/twitter_card.jpg" }
+      let(:client) { config.client }
+
+      it 'can upload successfully' do
+        response = client.upload_media(url)
+        response.should be_successful
+
+        response.data['media_id'].should be_a_kind_of Numeric
+        response.data['media_id_string'].should be_a_kind_of String
+      end
+
+      it 'uses the chunked upload endpoint', :aggregate_failures do

```

Comment:
We generally don't want to test internal implementation details. What we really care about testing is that we're able to upload large media.  According to [the docs](https://developer.twitter.com/en/docs/twitter-api/v1/media/upload-media/uploading-media/media-best-practices), the un-chunked upload path can handle:
- Images up to 5 MB
- Animated GIFs up to 15 MB
- No videos (all videos should use the chunked endpoint)

So if we test that we're able to successfully upload something larger than these limits, then I think we're good.  Right now, the `huge_pic.jpeg` file appears to be the only one that should break if we tried to upload it un-chunked, so that might be a good candidate for the chunked upload test.